compute_tools/src/compute.rs

```diff
@@ -298,7 +298,14 @@ impl ComputeNode {
             info!("Storage auth token not set");
         }
 
+        // Connect to pageserver
         let mut client = config.connect(NoTls)?;
+        let pageserver_connect_micros = Utc::now()
+            .signed_duration_since(start_time)
```

Comment:
Why use non-monotonic time here?

If `start_time` was an instant, there would be nothing to unwrap on R306.

Same goes for R357. I don't there is any reason for not using Instant here.